diagnostic.md

```diff
@@ -45,6 +61,11 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+weather = "sunny"; This assigns "sunny" to weather
+
+weather === "sunny"; This is saying/asking if it is the same in both
+value and type. Essentially identical. so FALSE
```

Comment:
Values would be 

```js
let weather; // undefined
weather = "sunny"; // "sunny"
weather === "sunny"; // true
```

reason being is that the VALUE of weather ("sunny") does infact strickly equal "sunny"